ui/src/common/components/Author.jsx

```diff
@@ -5,9 +5,25 @@ import { Map } from 'immutable';
 import { Tooltip } from 'antd';
 
 import AuthorAffiliationList from './AuthorAffiliationList';
-import { AUTHORS } from '../routes';
+import { AUTHORS, SUBMISSIONS_AUTHOR } from '../routes';
+import ExternalLink from './ExternalLink';
+import { getLinkToSearchWithBAI } from '../utils';
 
 class Author extends Component {
+  getCreateProfileTooltipMessage() {
```

Comment:
`render...` to indicate that it returns markup

Reply:
or alternatively you could refactor the whole thing, because it seems like "unlinked author with bai" logic is too much. So you can create a small function components here in this file (as different files but in a new folder called `Author`) `UnlinkedAuthor`, `LinkedAuthor`, `AuthorWithBai` and put the logic to decide which one render in the main `Author` component which will be exported.